week2/lc934_shortestBridge.js

```diff
@@ -0,0 +1,67 @@
+// Runtime: 120 ms, faster than 61.57% 
+// Memory Usage: 42 MB, less than 93.75%
+
+/**
+ * @param {number[][]} A
+ * @return {number}
+ */
+var shortestBridge = function(A) {
+    let queue = [];
+    let dx = [0, 0, 1, -1];
+    let dy = [1, -1, 0, 0];
+    let mark = false;
+    for(let i = 0; i < A.length; i++) {
+        if(mark) break;
+        for(let j = 0; j < A[0].length; j++) {
+            if(A[i][j] === 1) {
+                dfs(A, i, j, queue);

```

Comment:
Why dfs is better than bfs?

Reply:
I'm not sure too. Just because I checked the number of islands, my dfs version performance is better than bfs version, so I use dfs here.
![image](https://user-images.githubusercontent.com/17009429/88977021-f0202f80-d271-11ea-9e4c-872809dfdc2a.png)